composer.json

```diff
@@ -16,7 +16,8 @@
         "psr-4": {"Birke\\": "src/"}
     },
     "require": {
-        "paragonie/random_compat": "^1.1.4"
+        "paragonie/random_compat": "^2.0.2",
+        "predis/predis": "^1.1"
```

Comment:
Please move redis to a `suggest` section in composer. I don't want this package to depend on Redis.

Reply:
Hi, I use class `Predis\Client` in file `src/Rememberme/Storage/Redis.php` so it's a real dependency.
If you don't want this package to depend on Redis, you can reject this PR because the we can't add the redis storage.

Reply:
Yes, it's a real dependency of the storage adapter, but making it a `require` dependency, would make the whole package dependent on Redis, even if you use an alternative storage method. That's why I proposed using `suggest` instead of `require`. In projects where Redis is used, the dependency can be resolved because the project's `composer.json` can require redis. 

For an example of how other libraries do this, have a look at the monolog library: https://github.com/Seldaek/monolog There you have all kinds of services (elasticsearch, AWS, message queues) as suggested libraries, their class names referenced inside the program files, but not installed as dependencies by default. 

I think this is a similar case and would like to handle it similarly.